src/script/lua_api/l_async.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
```

Comment:
same

Reply:
In terms of the license header? I added it now.